docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -68,99 +73,111 @@ To develop and test the app, you'll install it on your [sandbox store](#configur
 | Sites & Routes | read-only | `store_sites_read_only` |
 | Storefront API Tokens | generate tokens | `store_storefront_api` |
 
-4. On the **App Supported Features** tab, indicate that this is a multi-storefront enabled app.
+4. On the **App Supported Features** tab, select the option that indicates this is a multi-storefront enabled app.
 
 5. Click **Update & Close**, then click **Confirm Update** to acknowledge the OAuth scopes you configured.
 
-Further steps in this guide require access to the Dev Portal, so keep it handy.
+Make sure to remain signed into the Dev Portal
 
 ## Configure Stripe
 
-We've designed this integration to use Stripe Connect so that your app can use one connection to submit payments to multiple merchants' previously configured Stripe accounts. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
+Subscription Foundation uses Stripe Connect so that your app can use one signle connection to submit payments to multiple previously configured Stripe merchant accounts. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
```

Comment:
```suggestion
Subscription Foundation uses Stripe Connect so that your app can use one single connection to submit payments to the previously configured Stripe accounts of multiple BigCommerce merchants. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
```

I want to be clear that the `merchants` are the same BigCommerce merchants we reference elsewhere, and that there isn't such a thing as a `Stripe merchant`.

What's the reason to ax the possessive? IMO, it's fewer words with clearer relationships, so I'm interested in the feeling or rule behind this pattern I'm seeing throughout.